README.md

```diff
@@ -5,14 +5,13 @@ Web service which collects and serves bug reports.
 To run it, do:
 
 ```
-BUGS_USER=<user> BUGS_PASS=<password> go run src/github.com/matrix-org/rageshake/main.go PORT
+go get github.com/constabulary/gb/...
+gb build
```

Comment:
FWIW I see no reason why this should be a GB project - we don't actually depend on anything that isn't in the stdlib. I don't expect you to change this now, but it may be nice to change this in the future.